CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Liana daemon and GUI release notes
 
+## 7.0
+
+This release introduces support for Wizardsardine servers as an optional backend for the Liana GUI
+and Electrum server as an optional node interface for the Liana daemon.
+
+### Breaking changes
+
+Liana Daemon has a migration for its database to add a new table `transactions` to keep in cache
+transactions to reduce requests to the node. Once Liana GUI or `lianad` run for the first time
+this migration is applied and cannot be reverted.
```

Comment:
A bit too many technical details IMO, we should just let the reader now what it means for them.

```suggestion
Running Liana v7 on an existing installation will migrate its database. Once migrated the database won't be compatible with previous versions of Liana. This means you won't be able to open with Liana v6 (or below) any wallet opened or created with Liana v7.
```